src/ip/Address.cc

```diff
@@ -101,6 +101,14 @@ Ip::Address::applyMask(Ip::Address const &mask_addr)
     return changes;
 }
 
+int
```

Comment:
on a deeper look, it appears that this methods return result is ignored by (void) in the only caller. So removing the return result entirely is appropriate. The applyMask() return is sometimes used - as an int, so that is broken but out of scope to fix here.

Reply:
I think we have a miscommunication here.

What I mean is that this new applyClientMask() method does not need to produce a return value, so do not produce one. Just call the relevant (void)applyMask(mask); and have no return values from applyClientMask().